exercises/practice/nucleotide-count/error-checking

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Synopsis:
+# Insert data from 'error_data.csv' into check constrained tables and save results to 'check_errors.csv'.
+# The csv can than be imported into the tests table to be shown to students.
+
+# Arguments:
+# $1: exercise slug
+
+# Output:
+# Writes the test results to a check_errors.csv file.
+
+slug=$1
+snake_slug=$(echo "${slug}" | sed 's/-/_/g')
```

Comment:
```suggestion
snake_slug="${slug//-/_}"
```

Reply:
I've been wondering why we kebab-snake names: `nucleotide-count_test.sql`. Should we not stick with one style?

Reply:
Yes, we should keep it one way.
We had one commit, where the case was changed, but only for the table name.
I could not find it, we should have them all the same.

Are there arguments for kebab or snake? I would prefer to have everything in snake case. But I have no real reason.

Reply:
I'm leaning towards using kebab everywhere so the slug and filenames match up. If we use snake, we need to juggle both, one of the slug, the other for the file. `exercises/practice/${kebab}/${snake}.sql`.